csuntechlab/iSTART
# STR-246 -- Refactor Input Comparison

# Description
Refactored the Vuex store, removed old artifact code not being used; Redid all validation on the Input Comparison template. **Also fixed top reset on refactor**
  
# Testing
1. Open any module and visit the InputComparison slide.
2. Check if form meets this criteria + weird combinations.
> When all inputs are valid, enable continue.
> When all inputs are invalid, disable continue.
> When input is not filled i.e (null undefined, empty string).
> When validated initially, then input a invalid input.
3. Ensure invalid input error appears when need be.
4. Make sure all modules with InputComparison work with above criteria.
